dandi/tests/test_pynwb_utils.py

```diff
@@ -63,3 +63,7 @@ def search(v):
         )
         == "2.1.0"
     )
+
+
+def test_nwbfile_to_dict(simple1_nwb):
+    assert nwbfile_to_metadata_dict(simple1_nwb) == {'attributes': {'namespace': 'core', 'neurodata_type': 'NWBFile', 'nwb_version': '2.2.2', 'object_id': 'f852e236-37db-4e2b-81ef-37f22ba20bb4'}, 'datasets': {'file_create_date': {'data': '<HDF5 dataset "file_create_date": shape (1,), type "|O">'}, 'identifier': {'data': 'identifier1'}, 'session_description': {'data': 'session_description1'}, 'session_start_time': {'data': '2017-04-15T12:00:00+00:00'}, 'timestamps_reference_time': {'data': '2017-04-15T12:00:00+00:00'}}, 'groups': {'general': {'datasets': {'experiment_description': {'data': 'experiment_description1'}, 'experimenter': {'data': '<HDF5 dataset "experimenter": shape (1,), type "|O">'}, 'institution': {'data': 'institution1'}, 'keywords': {'data': '<HDF5 dataset "keywords": shape (2,), type "|O">'}, 'lab': {'data': 'lab1'}, 'related_publications': {'data': '<HDF5 dataset "related_publications": shape (1,), type "|O">'}, 'session_id': {'data': 'session_id1'}}}, 'stimulus': {'groups': {}}}}
```

Comment:
@yarikoptic I've got a noob question for you. What am I doing wrong here? It looks like a string comparison but they should both be dicts? I'm confused.

Reply:
I figured out the problem. Fresh uuids are generated from each object_id, so these aren't matching. Maybe I can add a seed or something.

Reply:
sorry, just saw it.  If there was a way to seed it - would indeed be great. Otherwise I guess we would need to provide some helper to compare structs/dicts ignoring some fields (may be even assuring that they change ;)).